source/HailPanel.cpp

```diff
@@ -55,25 +55,22 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship)
 		message = "(There is no response to your hail.)";
 	else if(!hasLanguage)
 		message = "(An alien voice says something in a language you do not recognize.)";
+	// Update default hail responses based on the hailed ship's government and status condition.
 	else if(gov->IsEnemy())
 	{
-		if(ship->IsDisabled())
-			message = GameData::Phrases().Get("hostile disabled")->Get();
-		else
+		if(!ship->IsDisabled())
 		{
 			SetBribe(gov->GetBribeFraction());
 			if(bribe)
 				message = "If you want us to leave you alone, it'll cost you "
 					+ Format::Number(bribe) + " credits.";
 		}
 	}
-	else if(ship->IsDisabled())
+	else if(!gov->IsEnemy() && ship->IsDisabled())
```

Comment:
the `!gov->IsEnemy() && ` is obsolete again, it was only neccessary in the temporary state with the `if(gov->IsEnemy() && !ship->IsDisabled())` clause.